wait-for-broker.sh

```diff
@@ -0,0 +1,28 @@
+cmd="$@"
+
+if [ ! -z "$FLOWER_SKIP_BROKER_READY" ]; then
+	>&2 echo "Skipping broker readiness, executing command"
+	exec $cmd
+fi
+
+while true;
```

Comment:
infinite loops are usually not the best idea, there should be some execution limit, like 15 tries (with optional exponential backoff.